test/mongoGetCommand.test.ts

```diff
@@ -450,12 +450,11 @@ suite("scrapbook parsing Tests", () => {
         assert.deepEqual(err.range.start.character, 26);
     });
 
-    test("Trying to use pretty()", () => {
+    test("Chained command: to use pretty()", () => {
```

Comment:
Need more tests.  Be creative - try to break your code.  And what about all the testcases in the issues that have been filed?

Reply:
As a first iteration, I wanted feedback on the code initially (whether my approach is starkly different from what's expected). 

I'll add tests too now. :)

Reply:
Gotcha.  Saw nothing obviously wrong with the approach, although I'd need to play with it to determine how successful it is.

Reply:
How good is the error handling experience (both in the editor and trying to run commands)?

Reply:
Reply?

Reply:
I tried out the following cases: 
I miss the separating period -> Expected an EOF, rather than the second functionCall
Mid-typing: 
Have dot, not function call -> Expected a functionCall instead of ';'
Haven't entered brackets -> Expected '(', instead of ';'

Across commands: 
If there's no semi-colon, the error occurs at the start of the next command. 
If there's a semicolon, the error position is contained within current command. 

Running: 
It only runs if the command seems legit. There could be errors from the shell.
Eg. `db.people.find().limit()` would execute, but return a shell error . The limit function needs an argument.

Reply:
So do you think the user experience is acceptable?

Reply:
The behavior for errors across commands seems compliant with behavior elsewhere. 
Running the command has a fail-safe (the shell). 
Mid-type : The errors make sense to me. 

I think it is acceptable UX wise.